content/00.build/30.zksync-sso/10.architecture.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Architecture
+description: Learn about architectural components of ZKsync SSO and how they work together.
+---
+
+The ZKsync SSO consists of three components that work together to offer a seamless experience for both developers and users.
+
+## ZKsync SSO Developer SDK
+
+The **Developer SDK** is a client-side library that developers can integrate into their applications with minimal effort.
+It handles authentication, session management, and spend limit controls,
+allowing developers to enhance security and streamline the user experience, without dealing with complex wallet and key management infrastructure.
+
+**Features:**
+
+- **Authentication:** Enables users to authenticate using passkeys, supporting biometric methods like Face ID and fingerprint recognition.
+- **Session and Spend Limit Management:** Allows applications to manage user sessions with specific token limits,
+enhancing security by restricting the scope and usage of session keys.
```

Comment:
I don't think sessions enhance security neither for the developer nor for the user. Can you explain what exactly you mean here?

Reply:
I think the idea is that 'smart' sessions *could be* more secure than 'dumb' sessions. We should be clear that the sessions you made a clearly more secure for users **if** developers configure them correctly!